setup.py

```diff
@@ -17,19 +17,18 @@
         'License :: OSI Approved :: GNU Affero General Public License v3',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.5',
         'Framework :: Django',
-        'Framework :: Django :: 1.8',
-        'Framework :: Django :: 1.9',
-        'Framework :: Django :: 1.10',
-        'Framework :: Django :: 1.11',
+        'Framework :: Django :: 2.2.13',
+        'Framework :: Django :: 3.0',
+        'Framework :: Django :: 3.1',
     ],
     packages=[
         'edxlearndot',
         'tests'
     ],
     install_requires=[
-        "django >= 1.8, < 2.0",
+        "django>=2.2, >3, <3.1",
```

Comment:
This has to be:

```suggestion
        "django>=2.2,<3",
```

Otherwise, when you try to install this edx-learndot package in the edxapp venv, you get:

```
pip install -e /edx/src/edx-learndot
Looking in indexes: http://edx.devstack.devpi:3141/root/pypi/+simple/, https://pypi.python.org/simple
Obtaining file:///edx/src/edx-learndot
Collecting django<3.1,>3,>=2.2
  Downloading http://edx.devstack.devpi:3141/root/pypi/%2Bf/e16/30333248c9b3d/Django-3.0.7-py3-none-any.whl (7.5MB)
ERROR: Package 'Django' requires a different Python: 3.5.2 not in '>=3.6'
```

Reply:
Oh I didn't notice that, thanks for bringing this to my attention.